blocks/blogheader/blogheader.js

```diff
@@ -0,0 +1,25 @@
+import { getMetadata, decorateIcons } from '../../scripts/aem.js';
+
+export default async function decorate(block) {
+  const blogHeaderMeta = getMetadata('blogheader');
+  const blogHeaderPath = blogHeaderMeta
+    ? new URL(blogHeaderMeta).pathname
+    : '/blogs/blog-nav';
```

Comment:
let's please put it under `/blogs/fragments/blog-nav`. That way we can keep track of fragments in a designated place and have indexing configurations for them, like https://github.com/hlxsites/servicenow/blob/main/helix-query.yaml#L7 and later in `robots.txt` or in `metadata.xls`.

Also you could create a dictionary to map locale to blog nav
```javascript
const defaultBlogNavs = {
  'en-US': '/blogs/fragments/blog-nav',
  'fr-FR': '/fr/blogs/fragments/blog-nav',
  'de-DE': '/de/blogs/fragments/blog-nav',
 ...
}
```

then you can use the following function to retrieve the current locale
 https://github.com/hlxsites/servicenow/blob/main/scripts/scripts.js#L93 and take from the dictionary the correct contextual nav.
 
```javascript
  const blogHeaderPath = blogHeaderMeta
    ? new URL(blogHeaderMeta).pathname
    :  defaultBlogNavs[getLocale()] || '/blogs/blog-nav';
```

Reply:
Actually, sorry 🙂 I had a better idea in https://github.com/hlxsites/servicenow/pull/46 (which I needed to do for the bloglist as well). we can have these static per locale information globally in the `scripts.js`. That way, if they ever want to add a new locale, they can simply add a new entry locally with everything needed and don't have to modify each individual block JS.